### ISSUE TITLE
Broken against ember-cli 2.7.0-beta.2

### ISSUE THREAD

**Issue by tragiclifestories** [switchfly/ember-cli-mocha#116]
Reproduction steps:

```
npm i -g ember-cli@2.7.0-beta.2
ember new example && cd example
ember i ember-cli-mocha
=> Installed packages for tooling via npm...

...

ember test
=> DEPRECATION: ember-cli-test-loader should now be included as an NPM module with version 1.1.0 or greater.
=> Built project successfully. Stored in "/Users/jamesturley/Documents/personal/js/testest/tmp/core_object-tests_dist-ZjwXsRjx.tmp".
=> not ok 1 PhantomJS 2.1 - Global error: Error: Could not find module `ember-cli/test-loader` imported from `(require)` at http://localhost:7357/assets/vendor.js, line 221
    ---
        Log: |
            { type: 'error',
              text: 'Error: Could not find module `ember-cli/test-loader` imported from `(require)` at http://localhost:7357/assets/vendor.js, line 221\n' }
    ...
```

Looks to be something to do with ember-cli-test-loader moving over to NPM (see [here](https://github.com/ember-cli/ember-cli/pull/5709)). I'm assuming that what's required is something similar to https://github.com/ember-cli/ember-cli-qunit/pull/113.

**Comment by Turbo87** [comment]
this will be solved by https://github.com/switchfly/ember-cli-mocha/pull/111

**Comment by tragiclifestories** [comment]
So it will.

**Comment by rwjblue** [comment]
Published #111 as v0.10.3
